libs/ledger-live-common/src/exchange/swap/initSwap.ts

```diff
@@ -98,7 +102,14 @@ const initSwap = (input: InitSwapInput): Observable<SwapRequestEvent> => {
           });
 
           if (unsubscribed || !res || !res.data) return;
-        } catch (e) {
+        } catch (e: any) {
+          if (e.status == 422) {
```

Comment:
for now we only have this single error case associated to status code 422, but we might have others in the future.

I would rather test more specifically on the messageKey="WRONG_OR_EXPIRED_RATE_ID"